brave/brave-core
# Add VPN widget metrics

Resolves https://github.com/brave/brave-browser/issues/42515



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start with fresh profile, ensure no metrics exist.
1. Click on the "free trial" button on the VPN widget. Ensure the widget usage metric has the value 0.
2. Activate Brave VPN not using the widget, and then use the VPN connection toggle in the widget. Ensure the value is now 1.
3. Hide the widget, ensure the hide widget metric has the value 1.
